ClientLib/src/main/java/com/o3dr/android/client/apis/GimbalApi.java

```diff
@@ -169,7 +169,7 @@ public void onError(int error){
      * @param yaw       the desired gimbal yaw in degrees
      * @param listener Register a callback to receive update of the command execution state. Must be non-null.
      */
-    public void updateGimbalOrientation(double pitch, double roll, double yaw, final GimbalOrientationListener listener){
+    public void updateGimbalOrientation(float pitch, float roll, float yaw, final GimbalOrientationListener listener){
```

Comment:
@TShapinsky you also need to update the `params` setup below to use `putFloat` instead of `putDouble`.
Following that, the receiving end should be updated as well to take `float` instead of `double`.